tests/election.py

```diff
@@ -96,7 +96,7 @@ def run(args):
             LOG.debug("Stopping primary")
             primary.stop()
 
-            LOG.debug("Waiting for a new primary to be elected...")
+            LOG.debug(f"Waiting {max_election_duration} for a new primary to be elected...")
             time.sleep(max_election_duration)
```

Comment:
we should skip this sleep for pbft

Reply:
I've tried removing this locally and it seems to introduce a new failure case (ie - beyond the view change instability that should be fixed by #1001). Going to leave the sleep in for now, and we can take another look if this is still unnecessarily slow.